src/rdkafka_admin.h

```diff
@@ -283,6 +283,33 @@ struct rd_kafka_ConfigResource_result_s {
                               *   but with response error values. */
 };
 
+typedef enum rd_kafka_InternalConfigResourceType_t {
+        RD_KAFKA_INTERNAL_RESOURCE_CONFIG_UNKNOWN = 0,
+        RD_KAFKA_INTERNAL_RESOURCE_CONFIG_ANY     = 1,
+        RD_KAFKA_INTERNAL_RESOURCE_CONFIG_TOPIC   = 2,
+        RD_KAFKA_INTERNAL_RESOURCE_CONFIG_GROUP =
+            32,  // Changed value for config APIs
+        RD_KAFKA_INTERNAL_RESOURCE_CONFIG_BROKER = 4,
+        RD_KAFKA_INTERNAL_RESOURCE_CONFIG_CNT,
+} rd_kafka_InternalConfigResourceType_t;
```

Comment:
Let's do this conversion only for GROUP right now. We can think about others later.